gkeep_sync/sync_note.py

```diff
@@ -0,0 +1,163 @@
+import os
+import hashlib
+import json
+import requests
+
+from .utils import benchmark, traverse_files, DEFAULT_CONFIG_PATH
+
+SYNC_LABEL = 'autosync'
+FILE_EXTENSION = 'md'
+
+
+class GkeepSyncAPI:
+    def __init__(self, keep):
+        config = json.load(open(DEFAULT_CONFIG_PATH))
+        config['notes_root'] = os.path.expanduser(config['notes_root'])
+
+        self.config = config
+        self.keep = keep
+
+    def create_note(self, title, text):
+        print(f"Creating a note {title}")
+        newNote = self.keep.createNote(title, text)
+        label = self.keep.findLabel(SYNC_LABEL)
+        newNote.labels.add(label)
+
+        return newNote
+
+    def find_note(self, title):
+        label = self.keep.findLabel(SYNC_LABEL)
+        gnotes = list(self.keep.find(labels=[label], func=lambda note: note.title == title))
+
+        if len(gnotes) > 1:
+            raise Exception(f'Too many notes match the title: {title}')
+        return (gnotes[0] if len(gnotes) == 1 else None)
```

Comment:
No need for parentheses here. You can also change `len(gnotes) == 1` to just `gnotes` since you already know that length is eight 0 or 1

Reply:
I prefer the version with the parens.